packager.sh

```diff
@@ -709,7 +753,7 @@ if [ $BUILD_DEVEL -eq 1 ]; then
         pkg_deb="-0"
```

Comment:
at line 735 could we uncomment echo "pkg [$pkg_vers] [$pkg_maj] [$pkg_min] [$pkg_bfx] [$pkg_deb] [$pkg_suf]" to be consistent with 'hazardlib' if we decide to keep it uncommented